blog/introduction.md

```diff
@@ -148,7 +148,7 @@ $$a_{i}\cdot s * b_{i}\cdot s - c_{i}\cdot s = 3*3 -9 = 0$$
 
 </center>
 
-As you can see the computations are fine, so R1CS for circuit one is ok. Alice will now use a *Cricom* for generating a R1CS. First she need to creates a *Cricom* template file (*task.cricom*) which defines a constraints for our code:
+As you can see the computations are fine, so R1CS for circuit one is ok. Alice will now use a *Cricom* for generating a R1CS. First she need to creates a *Cricom* template file (*[task.cricom](https://github.com/bright/zk-snarks-with-substrate/blob/main/circom/task.circom)*) which defines a constraints for our code:
```

Comment:
zk-snark concept image links to m1 branch, this to main, I suggest link all images to one branch.

Reply:
Done